src/main/webapp/mentors.js

```diff
@@ -55,83 +110,180 @@ function createMentorElement(mentor) {
   mentorCard.className = 'card-holder card h-100 border-primary mb-3';
 
   const mentorCardBody = document.createElement('div');
-  mentorCardBody.className = 'card-body pb-0';
+  mentorCardBody.className = 'card-body pb-0 text-center d-flex flex-column';
```

Comment:
It's whack that we can't comment on unchanged lines. Can we fix the lint error on line 18? If we intend to use `getMentors` in place of `getMockMentors` in the future, that's fine, let's silence the rule to express that intention: Can add `// eslint-disable-line no-used-vars` to the end of that line.

Reply:
Guessing you are referring to no-unused-vars. Thanks so much for showing me how to work disable certain linting!